src/assets/i18n/pt-PT.json5

```diff
@@ -4403,20 +4427,20 @@
   // "mydspace.show.supervisedWorkspace": "Supervised items",
   "mydspace.show.supervisedWorkspace": "Itens supervisionados",
 
-  // "mydspace.status.archived": "Archived",
-  "mydspace.status.archived": "Depósito aceite",
+  // "mydspace.status.mydspaceArchived": "Archived",
+  "mydspace.status.mydspaceArchived": "Depósito aceite",
```

Comment:
`Arquivado`?

Reply:
From a user point of view I would mantain the proposed translation, but open to other view points :-)